iot-hub/Samples/device/TwinSample/TwinSample.cs

```diff
@@ -59,11 +61,19 @@ public async Task RunSampleAsync(TimeSpan sampleRunningTime)
 
         private async Task OnDesiredPropertyChangedAsync(TwinCollection desiredProperties, object userContext)
         {
+            var reportedProperties = new TwinCollection();
+
             Console.WriteLine("\tDesired property changed:");
             Console.WriteLine($"\t{desiredProperties.ToJson()}");
 
+            // For the purpose of this sample, we'll blindly accept all twin property write requests.
+            foreach (KeyValuePair<string, object> desiredProperty in desiredProperties)
+            {
+                Console.WriteLine($"Setting {desiredProperty.Key} to {desiredProperty.Value}.");
+                reportedProperties[desiredProperty.Key] = desiredProperty.Value;
+            }
+
             Console.WriteLine("\tSending current time as reported property");

```

Comment:
we'll need to update this log to say that the reported property patch now acks all the desired property update requests + sends the current time

Reply:
Reworded